cockatrice/src/remoteclient.cpp

```diff
@@ -150,6 +150,11 @@ void RemoteClient::loginResponse(const Response &response)
         if (resp.missing_features_size() > 0 && settingsCache->getNotifyAboutUpdates())
                 emit notifyUserAboutUpdate();
 
+        if (resp.server_features_size() > 0) {
+            for (int i = 0; i < resp.server_features_size(); ++i)
+                settingsCache->addServerFeatures(QString::fromStdString(resp.server_features(i)));
```

Comment:
Keep in mind the cache would be changed based on the server your connecting
to.  It's some what of a grey area in my opinion.  Settings cache is just
that, it's not something that gets permanently written and reused later on
the client side after client unloading.

On Thursday, December 31, 2015, Gavin Bisesi <notifications@github.com>
wrote:

> In cockatrice/src/remoteclient.cpp
> <https://github.com/Cockatrice/Cockatrice/pull/1609#discussion_r48658705>:
>
> > @@ -150,6 +150,11 @@ void RemoteClient::loginResponse(const Response &response)
> >          if (resp.missing_features_size() > 0 && settingsCache->getNotifyAboutUpdates())
> >                  emit notifyUserAboutUpdate();
> >
> > +        if (resp.server_features_size() > 0) {
> > +            for (int i = 0; i < resp.server_features_size(); ++i)
> > +                settingsCache->addServerFeatures(QString::fromStdString(resp.server_features(i)));
>
> Yeah, I'm not liking this being on settingsCache - that's for local
> configuration regardless of what server you're on
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/1609/files#r48658705>.
>